libs/util/shared/src/lib/utils/temporal-extent-union.spec.ts

```diff
@@ -1,7 +1,30 @@
 import { getTemporalRangeUnion } from './temporal-extent-union'
+import { DateService } from '../services'
+
+// Create a minimal DateService mock by casting
+const dateServiceMock = {
+  formatDate: (
+    date: Date | string,
+    options?: Intl.DateTimeFormatOptions
+  ): string => {
+    const dateObj = typeof date === 'string' ? new Date(date) : date
+    return options
+      ? dateObj.toLocaleDateString('en', options)
```

Comment:
Should we be more precise and use `'en-US'`? i don't know if it changes something.